tests/common.go

```diff
@@ -5131,3 +5131,68 @@ func GetPoolIDsFromVolName(volName string) ([]string, error) {
 	}
 	return poolUuids, err
 }
+
+// WaitTillEnterMaintenance Wait till Node Enters maintenance mode
+func WaitTillEnterMaintenanceMode(n node.Node) {
+	t := func() (interface{}, bool, error) {
+		if Inst().V.IsNodeInMaintenance(n) == true {
+			return nil, true, nil
+		}
+		return nil, false, fmt.Errorf("Not in Maintenance mode")
+	}
+
+	_, err := task.DoRetryWithTimeout(t, 20*time.Minute, 2*time.Minute)
+	log.FailOnError(err, fmt.Sprintf("Failed to Enter maintenence mode on node [%s]", n.Name))
+}
+
+// ExitMaintenanceMode waits till Node exits from Maintenance Mode
+func ExitFromMaintenanceMode(n node.Node) {
+	log.InfoD(fmt.Sprintf("Exiting maintenence mode on Node %s", n.Name))
+	t := func() (interface{}, bool, error) {
+		if err := Inst().V.ExitMaintenance(n); err != nil {
+			if Inst().V.IsNodeOutOfMaintenance(n) == true {
+				return nil, true, nil
+			}
+			return nil, true, err
+		}
+		return nil, false, nil
+	}
+	_, err := task.DoRetryWithTimeout(t, 15*time.Minute, 2*time.Minute)
+	log.FailOnError(err, fmt.Sprintf("Fail to exit maintenence mode on Node %s", n.Name))
+}
+
+// ExitNodeFromMaintenance waits till all Nodes to exit from Maintenance mode
+// Checks for all the Storage Nodes present in the Cluster, in case if any node is in Maintenance mode
+// Function will attempt bringing back the Node out of Maintenance
+func ExitNodesFromMaintenanceMode() {
+	Nodes := node.GetStorageNodes()
+	for _, eachNode := range Nodes {
+		if Inst().V.IsNodeInMaintenance(eachNode) {
+			ExitFromMaintenanceMode(eachNode)
+		}
+	}
+}
+
+// GetPoolsPresentOnNodes returns all pools present in the Nodes
+func GetPoolsDetailsOnNode(n node.Node) []*opsapi.StoragePool {
+	var poolDetails []*opsapi.StoragePool
+
+	if node.IsStorageNode(n) == false {
+		log.Errorf("Node [%s] is not Storage Node", n.Id)
+		return nil
+	}
+
+	Nodes := node.GetStorageNodes()
```

Comment:
Don't define var that is local to the function with Capital letter

Reply:
fixed